envoy/ssl/private_key/private_key.h

```diff
@@ -51,6 +51,12 @@ class PrivateKeyMethodProvider {
    */
   virtual bool checkFips() PURE;
 
+  /**
+   * Check whether the private key method initialized successfully.
+   * @return true if the private key method initialized successfully, false if not.
+   */
+  virtual bool checkInitialized() PURE;
```

Comment:
maybe name it as `initialized()` or `isAvailable()`

Reply:
ok, done, thanks!